presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -298,14 +310,17 @@ public Page getOutput()
         private final List<Optional<Block>> partitionConstants;
         private final PagesSerde serde;
         private final List<PageBuilder> pageBuilders;
+        private final boolean replicateAny;
```

Comment:
This is not "replicate first row". It is actually "replicate first row from every pipeline", but this is an implementation detail I'm not going to expose.